mllib/src/test/scala/org/apache/spark/ml/classification/MultilayerPerceptronClassifierSuite.scala

```diff
@@ -82,6 +83,49 @@ class MultilayerPerceptronClassifierSuite
     }
   }
 
+  test("strong dataset test") {
+    val layers = Array[Int](4, 5, 5, 2)
```

Comment:
Can you make this test faster by using a simpler network, e.g., by removing one of the middle layers?